modules/Administration/language/en_us.lang.php

```diff
@@ -762,6 +762,10 @@
     // OAuth
     'LBL_OAUTH_TITLE' => "OAuth Keys",
     'LBL_OAUTH' => "OAuth key management",
+    'LBL_OAUTH2_CLIENTS_TITLE' => "OAuth Clients",
```

Comment:
Why keep using double quotes? The standard is single quotes

Reply:
Thanks